lib/base/application.cpp

```diff
@@ -34,6 +34,10 @@
 
 using namespace icinga;
 
+#ifdef _WIN32
+const DWORD EXCEPTION_CODE_CXX_EXCEPTION = 0xe06d7363;
```

Comment:
I'd make this a `#define` (as we usually did).

Reply:
Or are the build-in ones consts, too?

Reply:
Not that I know. Just waited to change this because there also was a comment where the constant is used. Will push an updated version in a moment.